pkg/pubsub/pubsub_init.go

```diff
@@ -0,0 +1,87 @@
+package pubsub
+
+import (
+	"context"
+	"fmt"
+
+	psClient "github.com/open-policy-agent/gatekeeper/pkg/pubsub/client"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+func AddToManager(m manager.Manager) error {
+	r := newReconciler(m)
+	return add(m, r)
+}
+
+type Reconciler struct {
+	client.Client
+	*runtime.Scheme
+}
+
+func newReconciler(mgr manager.Manager) *Reconciler {
+	return &Reconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}
+}
+
+func add(mgr manager.Manager, r reconcile.Reconciler) error {
```

Comment:
If this is a controller, this is specific to Gatekeeper (as opposed to `gator`, etc.), and should go under `controllers/`